src/main/java/seedu/duke/Parser.java

```diff
@@ -0,0 +1,56 @@
+package seedu.duke;
+
+/**
+ * Class that parses the command given.
+ *
+ */
+public class Parser {
+
+    public Parser() throws DukeException {
+
+    }
+
+    /**
+     * Takes in the command and chooses what function should be used.
+     *

```

Comment:
To my knowledge, this extra space is a JavaDoc violation. It is troublesome as VSCode automatically does it for you. Would suggest setting up the checkstyles soon.